src/actors/objects/mod.rs

```diff
@@ -168,7 +168,7 @@ impl CacheItemRequest for FetchFileMetaRequest {
             .data_cache
             .compute_memoized(FetchFileDataRequest(self.clone()))
             .map_err(ObjectError::Caching)
-            .and_then(move |data| {
+            .and_then(move |data: Arc<ObjectFile>| {
```

Comment:
`ObjectFile` is probably a bad name, since it's rather a `CachedObject` or `ObjectCacheEntry`, but let's at least rename the variable to make it more clear. Then, you can also remove the type annotation again.

Reply:
+1 on renaming `ObjectFile`, maybe to `ObjectHandle`.  -0 on removing the type annotation.  It's super handy to be able to easily jump to the type declaration of this variable.  In functions it's normal to have explicit type declarations, a closure of more than a few lines is basically the same as a function.  Anyway, that's all my reasoning for this.